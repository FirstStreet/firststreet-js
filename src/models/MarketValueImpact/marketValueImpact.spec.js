const MarketValueImpact = require('./index');
const mviProperty = require('../../__mocks__/mviProperty');

let data = null;

describe('MarketValueImpact', () => {
  beforeEach(() => {
    data = new MarketValueImpact(mviProperty);
  ***REMOVED***);

  it('should initialize', () => {
    expect(data).toMatchSnapshot();
  ***REMOVED***);

  it('should have loss totals', () => {
    expect(data.lossTotals('kt', 2033)).toBeDefined();
  ***REMOVED***);
***REMOVED***);
