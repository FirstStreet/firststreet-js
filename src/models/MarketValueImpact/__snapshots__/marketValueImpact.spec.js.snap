// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarketValueImpact should initialize 1`] = `
MarketValueImpact {
  "data": Object {
    "FSID": 450350219571,
    "results": Array [
      Object {
        "floodData": Array [
          Object {
            "data": Array [
              Object {
                "type": "loss_total",
                "unit": "dollars",
                "value": 2008,
              ***REMOVED***,
            ],
            "year": 2018,
          ***REMOVED***,
          Object {
            "data": Array [
              Object {
                "type": "loss_total",
                "unit": "dollars",
                "value": 2033,
              ***REMOVED***,
            ],
            "year": 2033,
          ***REMOVED***,
        ],
        "floodID": "kt",
      ***REMOVED***,
    ],
  ***REMOVED***,
***REMOVED***
`;
